packages/expo/src/Notifications/ExponentNotifications.web.ts

```diff
@@ -18,6 +20,19 @@ export default {
   async dismissAllNotifications(): Promise<void> {},
   async cancelScheduledNotification(notificationId: string): Promise<void> {},
   async cancelAllScheduledNotifications(): Promise<void> {},
-  //   getBadgeNumberAsync(): Promise<void> {},
-  //   setBadgeNumberAsync(badgeNumber: number): Promise<void> {},
+  async getBadgeNumberAsync(): Promise<number> {
+    return currentBadgeNumber;
+  },
+  async setBadgeNumberAsync(badgeNumber: number): Promise<void> {
+    currentBadgeNumber = badgeNumber;
```

Comment:
Possibly write this to `AsyncStorage`.

Reply:
I think we can keep it as a local variable as it does not need to retrieved somewhere else. That would also keep it as simple as possible.